src/components/atoms/UserAvatar/UserAvatar.tsx

```diff
@@ -0,0 +1,17 @@
+import { Avatar } from "@mui/material";
+import styles from "./UserAvatar.module.scss";
+
+export * from "./UserAvatar";
+
+type userAvatarProps = {
+  label: string;
+  imageSrc?: string;
+};
+
+export const UserAvatar = ({ label, imageSrc }: userAvatarProps) => {
+  return (
+    <Avatar className={styles.avatarSize} alt={label} src={imageSrc}>
```

Comment:
A czy awatar nie ma właściwości heihgt/width - jeśli tak, to myślę, że możęmy pozby ć się stylowania z CSS modules na rzecz wpisania w  propsy tego komponentu z material UI

Reply:
Można w taki sposób ustawić wielkość awatara według twórcy: 
```
<Avatar
  alt="Remy Sharp"
  src="/static/images/avatar/1.jpg"
  sx={{ width: 56, height: 56 }}
/>
```

Reply:
Moim zdaniem tak jest czytelniej i w razie gdybyśmy chcieli jakąś responsywność zrobić to łatwiej ostylować :)